assets/panel/debugger.properties

```diff
@@ -470,6 +470,11 @@ sourceFooter.unblackbox.accesskey=b
 # with a blackboxed source
 sourceFooter.blackboxed=Blackboxed source
 
+# LOCALIZATION NOTE (sourceFooter.mappedSource): Text associated
+# with a mapped source.
```

Comment:
Nit on the second sentence of the comment: `%S is replaced by the source map origin`.